Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:latest
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
+
+EXPOSE 3003
+
+CMD ["npm","run","server"]
```

Comment:
It's a good idea to use the friendly "standard" startup command "npm start" Just makes it easier for people picking up your project to get up and running without having to read your package.json for the exact command.